.circleci/config.yml

```diff
@@ -604,4 +604,16 @@ workflows:
                 - master
     jobs:
       - build:
-          context: instances_test_env
\ No newline at end of file
+          context: instances_test_env
+  nightly:
+    triggers:
+      - schedule:
+          # should trigger every day at 9 PM UTC (12 AM Israel Time)
+          cron: "0 21 * * *"
```

Comment:
should be trigger at 3:00 or 4:00 (AMI creation is at 1:00)
and we want the maximum amount of time during the day before the machines are down (as long as the build finishes before people start working (around 8:00-9:00).

@ShahafBenYakir WDYT?

Reply:
cool

Reply:
```suggestion
          cron: "0 0 * * *"
```